packages/vscode-extension/src/debug/taskTerminal/baseTaskTerminal.ts

```diff
@@ -41,13 +40,17 @@ export abstract class BaseTaskTerminal implements vscode.Pseudoterminal {
     this.do()
       .then((res) => {
         const error = res.isErr() ? res.error : undefined;
-        this.stop(error);
+        this.stop(error).catch((error) => {
+          this.writeEmitter.fire(`${error?.message as string}\r\n`);
```

Comment:
Is this newly added logic?

Reply:
Yes, we need to add `catch` to fix eslint problem. And add a log to show the error happened in `stop`.

Reply:
I see. So this is also a bug fix for the missing error handling on `stop`.